avaframe/out3Plot/outAna1Plots.py

```diff
@@ -13,14 +13,18 @@
 import avaframe.out3Plot.outDebugPlots as outDebugPlots
 
 
-def showSaveTimeSteps(cfgMain, cfg, particlesList, fieldsList, solSimi, Tsave, relDict, outDirTest, simName):
+def showSaveTimeSteps(cfgMain, cfgSimi, particlesList, fieldsList, solSimi, Tsave, header, outDirTest, simHash, simDFrow):
```

Comment:
I suggest to make 2 functions out of this: show and save
+ docstring 
+ pytest the save